### ISSUE TITLE
Kubernetes storage object creation

### ISSUE THREAD

**Issue by petervo** [cockpit-project/cockpit#4430]
Quite a few kubernetes storage plugins take names of either endpoints or secrets as parameters when creating volumes. Do we want to provide a way for admins to create those objects from cockpit? Right now since we are focused on admin user stories, we don't do much in the way of interfaces for object creation. However these sort of endpoints and secrets is definitely something that only admins would be dealing with, does it make sense to include a way to at least add new ones, if not manage them somewhere?

**Comment by stefwalter** [comment]
Yup, I think so. It definitely fits in the bucket of taking an application and deploying it. Creating the necessary secrets, routes and so on. I think the same would apply to actions like creating ingress objects, mapping names, etc.
